Add sampling info to Tracing tasks (#3397)
istio/istio.io: content/boilerplates/trace-sampling-warn.md

> When you enable tracing, you may also want to set the sampling rate that Istio will use for tracing.

Reviewer: Warnings are a good use of boilerplates. Just be sure to include the markup needed for the callout in the boilerplate:
https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts
```suggestion
{{< warning >}}
When you enable tracing, consider setting Istio's sampling rate for tracing too.
```

Author: Hugo errors out when `{` characters appear as the first non-whitespace characters in boilerplate.  Rather than try to engineer a workaround, I have removed that boilerplate altogether.

Reviewer 2: There's probably an issue with using shortcodes in boilerplates, depending on when the boilerplates get expanded. I'll look into it.

> This can be accomplished via the `pilot.traceSampling` option. By default, the sampling rate is set to 1%.

Reviewer: ```suggestion
Set the sampling rate using the `pilot.traceSampling` option. The default sampling rate is set to 1%.
{{< /warning >}}
```

Author: done.
